techniques/flash/FLASH8.html

```diff
@@ -186,7 +186,7 @@ <h3>Programmatically adding a group name with the accessible name of radio butto
          
             <p>This example class can be initialized as follows: </p>
          
-         <pre xml:space="preserve">var myFieldSet = new FieldSet("Personal Details",  // the legend 
+         <pre>var myFieldSet = new FieldSet("Personal Details",  // the legend 
```

Comment:
all the naked `<pre> ... </pre>` blocks in this technique could do with inner `<code> ... </code>`